docs/commands.md

```diff
@@ -302,14 +300,18 @@ Do not launch packager while building.
 
 #### `--port [number]`
 
-> default: process.env.RCT_METRO_PORT || 8081,
+> default: process.env.RCT_METRO_PORT || 8081
 
 #### `--terminal [string]`
 
 > default: process.env.REACT_TERMINAL || process.env.TERM_PROGRAM
 
 Launches the Metro Bundler in a new window using the specified terminal path.
 
+#### `--task [string]`
+
+Run custom Gradle task instead of default "installDebug" where "debug" is default variant.
```

Comment:
This " where "debug" is default variant." makes me confused a bit. Why it's noted here?

Reply:
If the intent is for `task` to be `install` and `debug` is just `variant` that gets composed together to `installDebug`, then we should indicate that.